modyn/trainer_server/internal/grpc/trainer_server_grpc_servicer.py

```diff
@@ -26,15 +34,19 @@
 from modyn.trainer_server.internal.utils.trainer_messages import TrainerMessages
 from modyn.trainer_server.internal.utils.training_info import TrainingInfo
 from modyn.trainer_server.internal.utils.training_process_info import TrainingProcessInfo
-from modyn.utils import current_time_millis, dynamic_module_import
+from modyn.utils import current_time_millis, dynamic_module_import, grpc_connection_established
 
 logger = logging.getLogger(__name__)
 
+EMIT_MESSAGE_PERCENTAGES = [0.25, 0.5, 0.75]
```

Comment:
Let's keep this in modyn.utils and export it there. I think we can have this global across the package

Reply:
I agree, done in latest commit.